app/Repositories/Eloquents/PostRepository.php

```diff
@@ -266,4 +266,12 @@ public function voteBestAnswer($input)
             return false;
         }
     }
+    // insert records posts_tags table
+    public function createPostsTags($tags, $post_id)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 270</br>Variable "post_id" is not in valid camel caps format